tests/network-ovn

```diff
@@ -617,9 +617,8 @@ ovn_forward_tests() {
     lxc network set lxdbr0 ipv4.routes=192.0.2.0/24 ipv6.routes=2001:db8:1:2::/64 --project=default
 
     # Add a static route to SNAT address on uplink to OVN network's router (simulating BGP route advert to uplink).
-    ovnIPv4="$(lxc network get ovn-virtual-network volatile.network.ipv4.address)"
+    # XXX: only needed for IPv6 now (see https://github.com/ovn-org/ovn/issues/124)
```

Comment:
@simondeziel btw, LXD should probably be updated to use the new setting they added https://github.com/dceara/ovn/commit/1977e67b081129c89fe90cfba2f3dda1ada3fe71 for this fix.

Reply:
https://github.com/canonical/lxd/issues/12814